src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1784,7 +1811,19 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                 if (bodyBinder != null)
                 {
                     importChain = bodyBinder.ImportChain;
+
+#if DEBUG
+                    InMethodBinder? inMethodBinder;
+                    ConcurrentDictionary<IdentifierNameSyntax, int>? identifierMap;
+                    buildIdentifierMapOfBindIdentifierTargets(syntaxNode, bodyBinder, out inMethodBinder, out identifierMap);
```

Comment:
>out inMethodBinder, out identifierMap [](http://example.com/codeflow?start=86&length=37)

Consider using `out var` for these two parameters since we are just forwarding them.